grails-app/controllers/org/pih/warehouse/api/InventoryApiController.groovy

```diff
@@ -31,4 +38,24 @@ class InventoryApiController {
 
         render(status: 200)
     }
+
+    def getReorderReport(ReorderReportFilterCommand command) {
+        if (command.hasErrors()) {
+            throw new ValidationException("Invalid filters", command.errors)
+        }
+        List<ReorderReportItemDto> reorderReport = dashboardService.getReorderReport(command)
```

Comment:
I don't know if DashboardService is the place where this report should be placed, but this is where this report has always been, so I'm happy to discuss it